perf/auto-qual-test/deploy_latest_daily.sh

```diff
@@ -0,0 +1,35 @@
+git clone https://github.com/istio/tools
+cd tools/perf/istio-install
+helm init --client-only
+TARGET_VERSION=$1
+LATEST_BUILD=$(curl -L https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/release-$TARGET_VERSION-latest.txt)
```

Comment:
Doesn't look like this or HELMREPO_URL is used?

Reply:
Line 6 is quite long, but it does use LATEST_BUILD to define HELMREPO_URL, which is in turn used in the setup script at the end of that line.

Reply:
I suggest use `https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/$TARGET_VERSION-latest.txt)`

and pass in full version like `release-1.1`   or `master`